app/dashboard/tip_views.py

```diff
@@ -367,10 +367,26 @@ def send_tip_2(request):
         TemplateResponse: Render the submission form.
 
     """
+
+    username = request.GET.get('username', None)
     is_user_authenticated = request.user.is_authenticated
     from_username = request.user.username if is_user_authenticated else ''
     primary_from_email = request.user.email if is_user_authenticated else ''
 
+    user = {}
+    if username:
+        profiles = Profile.objects.filter(handle__icontains=username)
```

Comment:
we should do an exact match here to prevent similar names matching: i.e. I try to tip user `Dan` but theres a user named `DanBot`. Since there's no `order_by` clause we can't be sure which user will get returned. Better to use `iexact` here.

Reply:
Good catch, i already changed it! :smiley: